app/src/main/java/com/monkeyteam/chimpagne/ui/DetailScreenSheet.kt

```diff
@@ -52,11 +69,22 @@ fun DetailScreenSheet(event: ChimpagneEvent?, onJoinClick: () -> Unit = {}) {
               }
 
           Button(
-              onClick = onJoinClick,
+              onClick = enhancedOnJoinClick,
               modifier = Modifier.align(Alignment.CenterHorizontally).testTag("join_button")) {
                 Text(stringResource(id = R.string.find_event_join_event_button_text))
               }
         }
+    if (showDialog && context != null) {
+      popUpCalendar(
+          onAccept = {
+            createCalendarIntent(event)?.let { context.startActivity(it) }
+                ?: Toast.makeText(context, "Event can't be added to calendar", Toast.LENGTH_SHORT)
```

Comment:
Great Work! Maybe this Toast can be made as a StringResource ?

Reply:
done